Overview.src.html

```diff
@@ -674,8 +677,9 @@ <h4 id=terminology-headers>Headers</h4>
 <span title=concept-request-window>window</span>.
 
 <p>A <span title=concept-request>request</span> has an associated
-<dfn title=concept-request-target-browsing-context>target browsing context</dfn> (null or a
-<span data-anolis-spec=html>browsing context</span>). Unless stated otherwise it is null.
+<dfn title=concept-request-target-browsing-context data-export data-dfn-for=request>target browsing
+  context</dfn> (null or a <span data-anolis-spec=html>browsing context</span>). Unless stated
```

Comment:
Ok, got it. I should have looked at the examples; I read "Do not use newlines inside "inline" elements" as opposed to HTML's rules which force breaks inside the opening tag.